app/helpers/notify_views_helper.rb

```diff
@@ -22,6 +22,11 @@ def edit_subscription_link(subscription)
     notify_link(url, t(".edit_link_text"))
   end
 
+  def edit_subscription_link_job_alert_banner_location_ab_test(subscription)
```

Comment:
From the name of this method I would expect it to return an ab test, not a link.

Reply:
how about `job_alert_banner_location_ab_test_edit_subscription_link` ?